### PR TITLE
Feature/configmodel extensions

### PR DESCRIPTION
**Pull request by HaydarAk**
From the discussion with the colleagues working on the configuration model, following changes were requested:

- New class ids:GenericEndpoint, as subclass of ids:Endpoint for non-IDS specific endpoints with some additional properties to store endpoint configuration related information. Reusage of the ids:BasicAuthentication to enable adding authentication information for ids:GenericEndpoints.  GenericEndpoints are descriptive and generic (as the name suggests) and are not planned to be used for communication between IDS components (Connector / AppStore / Broker).

- AppRoute should point to ids:Endpoint and not ids:AppEndpoint. Allows to refer to any kind of endpoint inside a route.

- Obj.property from ids:AppRoute to ids:Resource to refer a given output of a route (e.g. a file) to a corresponding ids:Resource.

### CODE REVIEW

**Review by JohannesLipp — APPROVED** [review]
thank you

**Comment by clange** [comment]
@HaydarAk I have checked your two commits
* https://github.com/International-Data-Spaces-Association/InformationModel/pull/367/commits/beea57243ab135300ce2f4ae4d820d36eb86862e
* https://github.com/International-Data-Spaces-Association/InformationModel/pull/367/commits/03522b792ad726bfefcf46a9e6a64fa5d2954dfd
and they are fine, thank you!

With @JohannesLipp I agreed to assign this approval back to you, so that you can approve my remaining changes, which I made after your commits.

File: model/communication/AppRoute.ttl [hunk 1]
```diff
@@ -36,8 +38,25 @@ ids:AppRoute a owl:Class;
...
+         idsm:forProperty ids:appRouteClearingHouse;
+         idsm:relationType idsm:OneToMany ;
+    ];
+
 .
 
+ids:RouteStep a owl:Class;
+              rdfs:subClassOf ids:AppRoute;
+              rdfs:label "Route step"@en ;
+              rdfs:comment """
```
Comment (JohannesLipp) [inline model/communication/AppRoute.ttl:51]: Why is this a multi-line comment with triple """ ? The line break after "hold" looks confusing.

File: model/infrastructure/Catalog.ttl [hunk 2]
```diff
@@ -33,27 +33,65 @@ ids:ResourceCatalog
 .
 
 ids:ParticipantCatalog
-    rdfs:subClassOf ids:Catalog;
     a owl:Class;
+    rdfs:subClassOf ids:Catalog;
     rdfs:label "Participant Catalog"@en ;
-    rdfs:comment "Class that aggregates Participants which, as a whole, from a (distributable) Catalog."@en ;
+    rdfs:comment "Class that aggregates Participants, as a whole, from a (distributable) Catalog."@en ;
```
Comment (JohannesLipp) [inline model/infrastructure/Catalog.ttl:39]: Linguistic comment: I would remove those commas completely.

File: model/infrastructure/Catalog.ttl [hunk 3]
```diff
@@ -33,27 +33,65 @@ ids:ResourceCatalog
...
     ];
 .
 
 ids:ConnectorCatalog
-    rdfs:subClassOf ids:Catalog;
     a owl:Class;
+    rdfs:subClassOf ids:Catalog;
     rdfs:label "Connector Catalog"@en ;
-    rdfs:comment "Class that aggregates Connectors which, as a whole, from a (distributable) Catalog."@en ;
+    rdfs:comment "Class that aggregates Connectors, as a whole, from a (distributable) Catalog."@en ;
```
Comment (JohannesLipp) [inline model/infrastructure/Catalog.ttl:50]: Linguistic comment: I would remove those commas completely.

File: model/infrastructure/Catalog.ttl [hunk 4]
```diff
@@ -33,27 +33,65 @@ ids:ResourceCatalog
...
         idsm:forProperty ids:listedConnector;
         idsm:relationType idsm:OneToMany;
     ];
 .
 
+ids:BrokerCatalog
+    a owl:Class;
+    rdfs:subClassOf ids:ConnectorCatalog;
+    rdfs:label "Broker Catalog"@en ;
+    rdfs:comment "Class that aggregates Brokers, as a whole, from a (distributable) Catalog."@en ;
```
Comment (JohannesLipp) [inline model/infrastructure/Catalog.ttl:61]: Linguistic comment: I would remove those commas completely.

File: model/infrastructure/Catalog.ttl (hunk 3 above)
Comment (JohannesLipp) [reply model/infrastructure/Catalog.ttl:50]: Also, other places in this file

File: model/infrastructure/Catalog.ttl [hunk 5]
```diff
@@ -86,3 +124,17 @@ ids:listedConnector a owl:ObjectProperty;
     rdfs:comment "A Connector, that is part of a Connector catalog."@en;
 .
 
+ids:listedClearingHouse a owl:ObjectProperty;
+   rdfs:subPropertyOf dcat:dataset;
+   rdfs:domain ids:ClearingHouseCatalog;
+   rdfs:range ids:ClearingHouse;
+   rdfs:label "listed Clearing House"@en;
+   rdfs:comment "A Clearing House, that is part of a Clearing House catalog."@en;
```
Comment (JohannesLipp) [inline model/infrastructure/Catalog.ttl:132]: (Linguistic): Remove the comma.

**Review by JohannesLipp — CHANGES_REQUESTED** [review]
Minor, mostly linguistic, changes requested via comments. One question: Do we create one ConfigModel per AppStore/AppRessource/ClearingHouse or one-for-all? It looks like the latter, is this intended? Thanks!

File: model/infrastructure/Catalog.ttl (hunk 5 above)
Comment (HaydarAk) [reply model/infrastructure/Catalog.ttl]: Resolved in ca3bcf0

File: model/infrastructure/Catalog.ttl (hunk 4 above)
Comment (HaydarAk) [reply model/infrastructure/Catalog.ttl]: Resolved in 2164269

File: model/infrastructure/Catalog.ttl (hunk 3 above)
Comment (HaydarAk) [reply model/infrastructure/Catalog.ttl]: Done. See 2164269

File: model/communication/AppRoute.ttl (hunk 1 above)
Comment (HaydarAk) [reply model/communication/AppRoute.ttl]: Multiline comments in turtle are written using either """ or '''
See https://www.w3.org/TR/turtle/#turtle-literals

But you are right. The line breaks after """ look a bit confusing. Removed them in 52d2897

**Review by JohannesLipp — APPROVED** [review]
Thank you for improving all points I pointed out before. Thanks!

**Pull request merged**